notebooks/tel_and_site/subsys_req_ver/tma/SITCOM-1276_star_selector_pointing_tests.ipynb

```diff
@@ -0,0 +1,231 @@
+    "Vmag_max = 5 # maximum magnitude of the stars in the list\n",
+    "t0 = '2024-3-8 01:00:00' #in UTC, ideally the central time at which observations will take place\n",
+    "sep_lo = 3.3 #in degrees, minimum separation between stars\n",
+    "sep_hi = 3.7 #in degrees, maximum separation between stars\n",
+    "elevation_min = 45. #  minimum elevation for stars in the pair, NOTE that some might go below\n",
+    "                    # this elevation threshold as the sky rotates during the BLOCK"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "002fa1f5-4ecf-4dd7-86c5-054496185fe8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# select the site\n",
+    "pachon_site = EarthLocation(lat=-30.2407216*u.deg,lon=-70.736583*u.deg,height=2700*u.m)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "da8eedbd-c529-4f6d-854b-604f8f652bde",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# we use the Yale Bright Star Catalog\n",
+    "catalog_list = Vizier.find_catalogs('Yale')\n",
+    "print({k:v.description for k,v in catalog_list.items()})"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "283b5c5f-973b-4e4d-8287-64afce1eace5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "Vizier.ROW_LIMIT=-1\n",
+    "catalogs = Vizier.get_catalogs('V/50') #this the Yale Bright Star Catalog\n",
+    "table = catalogs[0]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0d73751a-e2c1-4105-aac8-cae8ed57096a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# select brightest stars\n",
+    "selection_vmag = (table['Vmag'] < Vmag_max)\n",
+    "print(len(table[selection_vmag]))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cfea78a4-a894-45cc-83c2-4dfaa7e437d2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ra = table[selection_vmag]['RAJ2000']\n",
+    "dec = table[selection_vmag]['DEJ2000']\n",
+    "stars_radec = SkyCoord(ra,dec,unit=(u.hourangle,u.deg),frame='fk5') #YBSC is in FK5 ref frame"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4583fa89-598a-4555-967c-f37f4050c474",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "time = Time(t0) #in UTC\n",
+    "stars_altaz = stars_radec.transform_to(AltAz(obstime=time,location=pachon_site))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f3e008f8-a11f-417a-96b5-456edfb08ee9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# select stars that are high enough\n",
+    "selection_alt = (stars_altaz.alt > elevation_min*u.deg)\n",
+    "print(len(stars_altaz.alt[selection_alt]))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "85b14159-0a9a-470c-a769-6e2a046fb222",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# print the list\n",
```

Comment:
Printing out a header of the different columns here (or maybe putting in a table) would be useful to the reader.